content/topics/code-comments/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+content_type: topic
+ready: true
+title: Code comments
+---
+
+## Understanding Effective Comments in Coding
+
```

Comment:
Heading levels are weird. Why are there 2 level 2 headings in a row? The "Understanding Effective Comments in Coding" section is empty. 

Maybe the title should say "Understanding Effective Comments in Coding"